generators/generator-base.js

```diff
@@ -102,14 +103,26 @@ const NO_SEARCH_ENGINE = searchEngineTypes.FALSE;
 const NO_MESSAGE_BROKER = messageBrokerTypes.NO;
 const NO_WEBSOCKET = websocketTypes.FALSE;
 
+const isWin32 = os.platform() === 'win32';
```

Comment:
Can you update also:

https://github.com/jhipster/generator-jhipster/blob/11726d3460f64a654f7838400aaa50cf92fa21d8/generators/generator-base.js#L2039-L2041
```
  getUserHome() {
    return process.env[isWin32 ? 'USERPROFILE' : 'HOME'];
  }
```

https://github.com/jhipster/generator-jhipster/blob/11726d3460f64a654f7838400aaa50cf92fa21d8/generators/generator-base.js#L2176-L2178
```
    if (!isWin32) {
      buildCmd = `./${buildCmd}`;
    }
```

https://github.com/jhipster/generator-jhipster/blob/11726d3460f64a654f7838400aaa50cf92fa21d8/generators/generator-base.js#L2202-L2204
```
    if (!isWin32) {
      buildCmd = `./${buildCmd}`;
    }
```

Reply:
Done